bin/manager_start.sh

```diff
@@ -27,26 +23,46 @@ JAVA_OPTS="-XX:MaxDirectMemorySize=$MAX_DIRECT_MEMORY
 
 MAIN="spark.jobserver.JobManager"
 
-MESOS_OPTS=""
-if [ $1 == "mesos-cluster" ]; then
-    MESOS_OPTS="--master $MESOS_SPARK_DISPATCHER --deploy-mode cluster"
-    appdir=$REMOTE_JOBSERVER_DIR
-fi
+# copy files via spark-submit and read them from current (container) dir
+if [ $2 = "cluster" -a -z "$REMOTE_JOBSERVER_DIR" ]; then
+  SPARK_SUBMIT_OPTIONS="$SPARK_SUBMIT_OPTIONS
+    --master $1 --deploy-mode cluster
+    --conf spark.yarn.submit.waitAppCompletion=false
```

Comment:
The default is `waitAppCompletion=true` and means that the client write the current application status (running) every second into the log. Setting this to `false` mean, create the app and disconnect after the request has been accepted.